src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md

```diff
@@ -0,0 +1,138 @@
+| `aria-hidden="true"` | `.pf-c-notification-drawer__group-toggle-icon > *`, `.pf-c-notification-drawer__list-item-header-icon > *` | Hides icon for assistive technologies. |
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-notification-drawer` | `<div>` | Initiates the notification drawer. |
+| `.pf-c-notification-drawer__header` | `<div>` | Initiates the notification drawer header. |
+| `.pf-c-notification-drawer__header-title` | `<h1>` | Initiates the notification drawer header title. |
+| `.pf-c-notification-drawer__header-status` | `<span>` | Initiates the notification drawer header status. |
+| `.pf-c-notification-drawer__header-action` | `<div>` | Initiates the notification drawer header action. |
+| `.pf-c-notification-drawer__body` | `<div>` | Initiates the notification drawer body. |
+| `.pf-c-notification-drawer__list` | `<ul>` | Initiates a notification list. |
+| `.pf-c-notification-drawer__list-item` | `<li>` | Initiates a notification list item. |
+| `.pf-c-notification-drawer__list-item-header` | `<div>` | Initiates a notification list item header. |
+| `.pf-c-notification-drawer__list-item-header-icon` | `<span>` | Initiates a notification list item header icon. |
+| `.pf-c-notification-drawer__list-item-header-title` | `<h2>` | Initiates a notification list item header title. |
+| `.pf-c-notification-drawer__list-item-action` | `<div>` | Initiates a notification list item action. |
+| `.pf-c-notification-drawer__list-item-description` | `<div>` | Initiates a notification list item description. |
+| `.pf-c-notification-drawer__list-item-timestamp` | `<div>` | Initiates a notification list item timestamp. |
+| `.pf-c-notification-drawer__group` | `<section>` | Initiates a notification group. |
+| `.pf-c-notification-drawer__group-toggle` | `<button>` | Initiates a notification group toggle. |
+| `.pf-c-notification-drawer__group-title` | `<div>` | Initiates a notification group toggle title. |
+| `.pf-c-notification-drawer__group-count` | `<div>` | Initiates a notification group toggle count. |
+| `.pf-c-notification-drawer__group-icon` | `<span>` | Initiates a notification group toggle icon. |
+| `.pf-m-info` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the info state. |
+| `.pf-m-warning` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the warning state. |
+| `.pf-m-danger` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the danger state. |
+| `.pf-m-success` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the success state. |
+| `.pf-m-read` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the read state. |
+| `.pf-m-clickable` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item to indicate that it is clickable. |
```

Comment:
should this be `pf-m-selectable` instead?

Reply:
I wondered that, and if we might have a "selectable" state in the future that might be separate from what we're offering here, which is to indicate that the item is clickable. The way I envision it currently in the react component is the item can attach click events to it like, on click, add `.pf-m-read`, or execute a custom callback function. That's what this class adds are styles to say "you can click this and something will happen."

And I wonder if something could be both selectable and clickable, with different styles. Maybe `pf-m-hoverable`? We use that in other places to indicate hover styles to say something is interactive.